playbooks/robusta_playbooks/autoscaler.py

```diff
@@ -69,10 +69,7 @@ def alert_on_hpa_reached_limit(event: HorizontalPodAutoscalerChangeEvent, action
     if hpa.status.desiredReplicas != hpa.spec.maxReplicas:
         return  # didn't reached max replicas limit
 
-    avg_cpu = int(
-        hpa.status.currentCPUUtilizationPercentage
-        / (hpa.status.currentReplicas if hpa.status.currentReplicas > 0 else 1)
-    )
+    avg_cpu = int(hpa.status.currentCPUUtilizationPercentage)
```

Comment:
@Avi-Robusta 
This value isn't exactly avg cpu usage. 
From the docs:
```
current average CPU utilization over all pods, represented as a percentage of requested CPU, e.g. 70 means that an average pod is using now 70% of its requested CPU.
```

Do you think it will help to change the text to something like:
`On average, pods scaled under this HPA are using 75% of the requested cpu`

@aantn thoughts?

Reply:
@arikalon1 yes, I like that text. Super clear to me. @Avi-Robusta can we update it?

Reply:
on it